tests/integration/security/citadel/namespace_targeting_test.go

```diff
@@ -109,7 +107,7 @@ func runNamespaceTargetingTest(
 
 	secretGenerated := err == nil
 	if targeted != secretGenerated {
-		t.Errorf("expected secret generation for secret %s: %t, but got: %t",
+		ctx.Fatalf("expected secret generation for secret %s: %t, but got: %t",
```

Comment:
what's the reasoning to switch this from `Errorf` to `Fatalf`?

Reply:
The difference is fatal will stop the test and error will continue. In this case it doesn't really matter since its at the end of the test anyways

Reply:
:+1: